### PR TITLE
Add changes to not invoke apply settings after setting WPS Soft Push Button 

### PR DESCRIPTION
**Pull request by sancharbanerjee**
Executing wifi apply setting when triggering WPS Push Button from Data Model will reload the radio and thus WPS functionality will fail.

### CODE REVIEW

**Comment by sancharbanerjee** [comment]
@shilpa24balaji @sadhyama Please let me know if anything else needs to be done here.

**Comment by shilpa24balaji** [comment]
@sancharbanerjee I will check with our wifi team also to be sure before I merge this.

**Review by shilpa24balaji — APPROVED** [review]

**Review by sadhyama — APPROVED** [review]

**Comment by shilpa24balaji** [comment]
Sanjay Mullamuri  12:45 AM
We only need ApplySettings for configuration
This is a software trigger for push button, we do not need Applysettings, you can remove it.

**Pull request merged**
